src/routes/OrganizationsList/CreateOrganizationModal.tsx

```diff
@@ -135,14 +136,15 @@ export const CreateOrganizationModal = (
   };
 
   const createOrganizationHandler = async () => {
-    // handleModalToggle(); // check if this is needed
-    await fetch(`${quayAuth.QUAY_HOSTNAME}/api/v1/organization/`, {
-      method: 'POST',
-      headers: {
-        Authorization: `Bearer ${quayAuth.QUAY_OAUTH_TOKEN}`,
-      },
-      body: JSON.stringify({name: organizationName, email: organizationEmail}),
-    }).then;
+    const newOrgUrl = '/api/v1/organization/';
+    try {
+      await axios.post(newOrgUrl, {
```

Comment:
Shouldn't the actual api call live under `/resources`?

Reply:
aah yes you are correct